docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -313,23 +313,49 @@ go get github.com/hazelcast/hazelcast-go-client
 +
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
+
+import (
+  "context"
+  "fmt"
+  "log"
 
-func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+  "github.com/hazelcast/hazelcast-go-client"
+)
 
-  cb.Cluster().SetName("hello-world") <1>
+func main() {
+  ctx := context.TODO()
 
-  client, err := hazelcast.StartNewClientWithConfig(cb) <2>
+  cb := hazelcast.NewConfig()
```

Comment:
Can you use "hazelcast.Config{}` instead of `hazelcast.NewConfig()`. And also, can you rename `cb` to something else, maybe `config` or `cfg`?